libs/designsystem/src/lib/components/progress-circle/progress-circle-ring.component.ts

```diff
@@ -9,11 +7,16 @@ import { ThemeColor } from '../../helpers/theme-color.type';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ProgressCircleRingComponent {
+  readonly STROKE_MAP = {
```

Comment:
Consider marking this as `private` if it is only meant to be used in the `strokeWidth()` function 😄

Reply:
Just saw 45baadb - never mind this then 👀